src/components/MenuBar/ButtonBar.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+
+import classNames from 'classnames';
+import { makeStyles } from '@material-ui/core/styles';
+
+import LinkNoStyle from '../LinkNoStyle/LinkNoStyle';
+
+const useStyles = makeStyles(theme => {
+	const menuBarAdaptiveThreshold = theme.breakpoints.values.sm * 1.3;
+	return {
+		btn: {
+			fontFamily: theme.typography.fontFamily,
+			fontWeight: 500
+		},
+		borderBtn: {
+			margin: 10,
+			transition: theme.transitions.create(['color', 'background'], {
+				duration: theme.transitions.duration.complex
+			}),
+			'&:hover': {
+				color: '#FFFFFF',
+				background: '#DB99FD'
+			},
+			[theme.breakpoints.down(menuBarAdaptiveThreshold)]: {
+				// mobile
+				margin: 0
+			}
+		}
+	};
+});
+
+function ButtonBar({ isMobile }) {
+	const classes = useStyles();
+	/* eslint-disable */
+	const PoppinLink = ({ to, ...props }) =>
+		<LinkNoStyle to={to}>
+			<Button fullWidth={isMobile} className={classes.btn} {...props} />
+		</LinkNoStyle>;
+
+	const BorderLink = ({ to, ...props }) =>
+		<LinkNoStyle>
+			<Button variant='outlined' fullWidth={isMobile} className={classNames(classes.btn, classes.borderBtn)} {...props} />
+		</LinkNoStyle>;
+	/* eslint-enable */
+
+	const links = [
+		{
+			name: 'Home',
+			to: '/'
+		},
+		{
+			name: 'Schedule',
+			to: '/'
+		},
+		{
+			name: 'Prizes',
+			to: '/'
+		},
+		{
+			name: 'Gallery',
+			to: '/'
+		}
+	];
+
+	return (
+		<>
+			{links.map((link, index) =>
+				<PoppinLink to={link.to} key={`nav-${index}`}>
+					{link.name}
+				</PoppinLink>)}
+			<BorderLink to='/'>
+				Apply
+			</BorderLink>
```

Comment:
Maybe we would want a way to make this show up as “disabled” when we don’t have an active application going on right now? Maybe we could use Material-UI’s [outlined buttons](https://material-ui.com/components/buttons/#outlined-buttons) instead?

Reply:
We are already using the material-ui outlined buttons, I will update it so that the button is disabled for now